jetty-http/src/main/java/org/eclipse/jetty/http/HttpCookie.java

```diff
@@ -92,6 +127,8 @@ public HttpCookie(String setCookie)
         _comment = cookie.getComment();
         _version = cookie.getVersion();
         _expiration = _maxAge < 0 ? -1 : System.nanoTime() + TimeUnit.SECONDS.toNanos(_maxAge);
+        // TODO support for SameSite values has not yet been added to java.net.HttpCookie
```

Comment:
Not really a `TODO`, just a comment explaining why we do that. Just remove "TODO" and leave the comment.